content/400-reference/200-api-reference/300-environment-variables-reference.mdx

```diff
@@ -0,0 +1,151 @@
+---
+title: 'Environment variables'
+metaTitle: 'Prisma environment variables reference'
+metaDescription: 'This page gives an overview of all environment variables avalaible for use.'
+tocDepth: 2
+---
+
+## Overview
+
+This document describes different environment variables and their use cases.
+
+---
+
+## Prisma Client
+
+### DEBUG
```

Comment:
Can you add `<inlinecode>DEBUG</inlinecode>` to these headings? It renders them out like this: https://www.prisma.io/docs/reference/api-reference/prisma-schema-reference#datasource